src/ux/progress.ts

```diff
@@ -45,19 +45,24 @@ export class Progress extends UxBase {
     payload: Progress.Payload = {},
     options: Partial<Progress.Options> = Progress.DEFAULT_OPTIONS
   ): void {
-    const opts = Object.assign(Progress.DEFAULT_OPTIONS, options);
-    opts.format = util.format(opts.format, opts.title);
-    this.bar = CliUx.ux.progress({
-      format: opts.format,
-      barCompleteChar: opts.barCompleteChar,
-      barIncompleteChar: opts.barIncompleteChar,
-      linewrap: opts.linewrap,
-    }) as Progress.Bar;
-
-    this.bar.setTotal(total);
+    if (this.started) return;
+    this.started = true;
 
     this.maybeNoop(() => {
-      this._start(total, payload);
+      const opts = Object.assign(Progress.DEFAULT_OPTIONS, options);
+      opts.format = util.format(opts.format, opts.title);
+      this.bar = CliUx.ux.progress({
+        format: opts.format,
+        barCompleteChar: opts.barCompleteChar,
+        barIncompleteChar: opts.barIncompleteChar,
+        linewrap: opts.linewrap,
+      }) as Progress.Bar;
+
+      this.bar.setTotal(total);
+      this.bar.start(total);
+      if (Object.keys(payload)) {
```

Comment:
This should use `.length`
![image](https://user-images.githubusercontent.com/1084688/166574611-9d93eea2-6a63-4e51-a147-7566611049f0.png)

```suggestion
      if (Object.keys(payload).length) {
```